containers/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build the container project root directory in this sequence:
```

Comment:
Rather then depending on the host's npm why don't we just do everything in containers?
An unoptimized build stage could be something like:
```Dockerfile
FROM registry.fedoraproject.org/fedora-minimal:37 AS BUILD
USER root
RUN microdnf update -y && microdnf install -y nodejs
ADD . /sc
WORKDIR /sc
RUN npm install && npm run build
```

Followed by copying the files into the final container:
```Dockerfile
COPY --from=BUILD /var/cache/yum /var/cache/yum
COPY --from=BUILD /sc/dist/src /smee-client
COPY --from=BUILD /sc/package.json /smee-client/
```

Reply:
@scrufulufugus, let me know what you think.